### DISCUSSION TITLE
question to PatheticMustan or anyone else who knows

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#801] [(D) Developer]
@PatheticMustan or anyone else who knows about this? is it okay for me to shout out these hacks? (i have 250 subs) if so is there any credits i need to put in the description, or any where else i just really want to grow the hacks  and spread awareness to prodigy (because i know that's the goal in the first place) also is it okay to do giveaways of hacked prodigy accounts or no, if you could let me know about those 2 things that would be great, thx!

**Comment by PatheticMustan** [comment]
As long as you [credit us properly](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/meta/YOUTUBE_ATTRIBUTION.md), it should be fine.

**Comment by SakuraKitsunee** [comment]
okay thanks!
